cadquery/occ_impl/assembly.py

```diff
@@ -286,6 +287,71 @@ def _loc2vtk(
     return trans, (rot[1], rot[2], rot[0])
 
 
+def toVTKAssy(
+    assy: AssemblyProtocol,
+    color: Tuple[float, float, float, float] = (1.0, 1.0, 1.0, 1.0),
+    tolerance: float = 1e-3,
+    angularTolerance: float = 0.1,
+) -> vtkAssembly:
+
+    rv = vtkAssembly()
+
+    for shape, _, loc, col_ in assy:
+
+        col = col_.toTuple() if col_ else color
+
+        trans, rot = _loc2vtk(loc)
+
+        data = shape.toVtkPolyData(tolerance, angularTolerance)
+
+        # extract faces
+        extr = vtkExtractCellsByType()
+        extr.SetInputDataObject(data)
+
+        extr.AddCellType(VTK_LINE)
+        extr.AddCellType(VTK_VERTEX)
+        extr.Update()
+        data_edges = extr.GetOutput()
+
+        # extract edges
+        extr = vtkExtractCellsByType()
+        extr.SetInputDataObject(data)
+
+        extr.AddCellType(VTK_TRIANGLE)
+        extr.Update()
+        data_faces = extr.GetOutput()
+
+        # remove normals from edges
+        data_edges.GetPointData().RemoveArray("Normals")
+
+        # add both to the vtkAssy
+        mapper = vtkMapper()
+        mapper.AddInputDataObject(data_faces)
+
+        actor = vtkActor()
+        actor.SetMapper(mapper)
+        actor.SetPosition(*trans)
+        actor.SetOrientation(*rot)
+        actor.GetProperty().SetColor(*col[:3])
+        actor.GetProperty().SetOpacity(col[3])
+
+        rv.AddPart(actor)
+
+        mapper = vtkMapper()
+        mapper.AddInputDataObject(data_edges)
+
+        actor = vtkActor()
+        actor.SetMapper(mapper)
+        actor.SetPosition(*trans)
+        actor.SetOrientation(*rot)
+        actor.GetProperty().SetColor(0, 0, 0)
```

Comment:
Add option to apply style to edges?

I experimented with```        actor.GetProperty().SetOpacity(0)```
to achieve similar behavior of the CAD Assistant "Smoothly Shaded" option.

![image](https://github.com/user-attachments/assets/313eba40-f3c4-4cca-a151-590fca8c6ffa)

Reply:
I added something for this use case.

```python
show(style(box(1,1,1), edges=False))
```

![image](https://github.com/user-attachments/assets/bf848a34-5342-426e-9edb-9fc6a50139b3)

Reply:
Edges and mesh can be controlled separately now:
![image](https://github.com/user-attachments/assets/057484b3-a171-4c0a-a45d-ae31c413efbb)